File: 백준/Bronze/29163. Счастье Мистера Бина/Счастье Мистера Бина.cc
#include <iostream>
using namespace std;
int main(void) {
    int n, num, even = 0, odd = 0;
    cin >> n;
    for (int i = 0; i < n; i++) {
        cin >> num;
        if (num % 2 == 0)
            even += 1;
        else
            odd += 1;
    }
    cout << (even > odd ? "Happy" : "Sad");
    return 0;
}

File: 백준/Bronze/29163. Счастье Мистера Бина/Счастье Мистера Бина.py
import sys
input=lambda:sys.stdin.readline().rstrip()
n=int(input())
arr=[*map(int,input().split())]
even,odd=0,0
for i in range(n):
    if arr[i]%2==0:
        even+=1
    else:
        odd+=1
print("Happy" if even>odd else "Sad")